sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -641,11 +661,14 @@ public ReadFiles withProjection(Schema projectionSchema, Schema encoderSchema) {
 
     /** Specify Hadoop configuration for ParquetReader. */
     public ReadFiles withConfiguration(Map<String, String> configuration) {
+      checkArgument(configuration != null, "configuration can not be null");
       return toBuilder().setConfiguration(SerializableConfiguration.fromMap(configuration)).build();
     }
 
-    public ReadFiles withConfiguration(SerializableConfiguration configuration) {
```

Comment:
This change is to fix the API that was accidentally exposed as part of [BEAM-11861](https://issues.apache.org/jira/browse/BEAM-11861) #14078

Reply:
This was added in the ongoing version and it has not been released so we don't need to deprecate it, that's the reason why I wanted this fixed before the branch cut.